setup.py

```diff
@@ -60,13 +73,14 @@ def package_vars(version_file):
         "License :: CC0 1.0 Universal (CC0 1.0) Public Domain Dedication",
         # Specify the Python versions you support here. In particular, ensure
         # that you indicate whether you support Python 2, Python 3 or both.
-        "Programming Language :: Python :: 2",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
What about 3.11?

Reply:
I'm fine with adding it to the list. When `setup.py` was updated we had not added Python 3.11 support, but the `build.yml` workflow I pulled in does Python 3.11 testing.

Reply:
Please see [6319537](https://github.com/cisagov/cyhy-feeds/pull/43/commits/6319537536141086d853241f6602c3ff764b83bf).